services/frontend/src/components/WeatherForecast.js

```diff
@@ -0,0 +1,71 @@
+import WeatherWidgetContainer from "./WeatherWidgetContainer";
+import Header from "./Header";
+import { useState } from "react";
+import PropTypes from "prop-types";
+
+export const WeatherForecast = ({ locationData, defaultExpanded }) => {
+  // Placeholder weather data
+  const weatherData = [
```

Comment:
@lzkelley this is an example of static data that will be replaced with an API call once we have an endpoint.

Reply:
Cool.  I assume these would be different days (i.e. today + next 4 days)?

Reply:
Exactly. The final data structure will probably be more complicated than this, and I should include some kind of date/day notation above each forecast widget.